app/controllers/clearance/passwords_controller.rb

```diff
@@ -88,18 +87,19 @@ def flash_failure_after_update
       default: t('flashes.failure_after_update'))
   end
 
-  def forbid_missing_token
-    if params[:token].to_s.blank?
+  def check_for_forbidden_access
```

Comment:
Some history on the `forbid` / `forbidden` words: https://robots.thoughtbot.com/forbidden-kisses-http-fluency-in-clearance

Used to be a 403 when these things happened.